### DISCUSSION TITLE
I would like to suggest a board that will handle a Delta

### DISCUSSION THREAD

**Discussion by Loadmaster1** [FYSETC/FYSETC-E4#4] [Ideas]
I purchase the FYSETC E4 thinking that I could set it up as a replacement for my delta printer (Tevo Little Monster).  I like the E4, but it doesn't have enough to handle the delta.  The Little Monster has optical end stops that some boards provide a 3 pin connect that supplies voltage and the signal input.  

Something that you guys should consider in the future.

**Comment by GerogeFu** [comment]
Yes , we are already taking that into consideration, thanks for your suggestion.
